backend/copr-backend.spec

```diff
@@ -108,6 +108,8 @@ Requires(post): systemd
 Requires(preun): systemd
 Requires(postun): systemd
 
+Requires(pre): group(lighttpd)
```

Comment:
Don't we need the %{?fedora:} condition here?

Reply:
Why?
We will always need a package that provides this group.

Reply:
I actually had to install this to make the release happen :-( ee171546d58c7d21cf0a9e57962a049753df703c